pkg/v10/configmap/desired.go

```diff
@@ -96,6 +101,23 @@ func (s *Service) hasLegacyIngressController(ctx context.Context, releaseName st
 	return false, nil
 }
 
+func clusterAutoscalerValues(configMapValues ConfigMapValues) ([]byte, error) {
+	values := ClusterAutoscaler{
+		Cluster: ClusterAutoscalerCluster{
+			ID: configMapValues.ClusterID,
```

Comment:
This is all we should really need - with the discovery this should be fine.

Reply:
Nice. Good the config is so minimal.